lang/main.json

```diff
@@ -439,7 +439,23 @@
         "streamKey": "Live stream key",
         "thankYou": "Thank you for using {{appName}}!",
         "token": "token",
-        "tokenAuthFailed": "Sorry, you're not allowed to join this call.",
+        "tokenAuthFailed": "Sorry, you're not allowed to join this call. Possible reasons: {{reason}}",
+        "tokenAuthFailedReason": {
+            "audInvalid": "Ivalid `aud` value. It should be `jitsi`.",
+            "contextNotFound": "The `context` object is missing from the payload.",
+            "expInvalid": "Invalid `exp` value.",
+            "featureInvalid": "Invalid feature",
+            "featuresNotFound": "The `features` object is missing from the payload.",
+            "featureValueInvalid": "Invalid value for feature",
+            "headerOrPayload": "Missing the header or payload.",
+            "issInvalid": "Invalid `iss` value. It should be `chat`.",
```

Comment:
Same as above, "chat" is a JaaS convention. Maybe we can tell users to check "the documentation" so it's generic enough?

Reply:
Sorry I hadn't notied this is inside an if (jaas) check so it's ok.